kalitectl.py

```diff
@@ -592,6 +605,85 @@ def status():
 }
 
 
+def diagnose():
+    """
```

Comment:
I guess worst case is they get a failure with traceback on the command line when running it.